### PR TITLE
Part fail details report client

### PR DESCRIPTION
**Pull request by atcampbell**

### CODE REVIEW

File: src/Service.Host/client/src/components/Root.js [hunk 1]
```diff
@@ -371,6 +373,16 @@ const Root = ({ store }) => (
                                         path="/production/quality/part-fails/create"
                                         component={CreatePartFail}
                                     />
+                                    <Route
+                                        exact
+                                        path="/production/quality/part-fails/detail-report/report"
```
Comment (andrewfraser73) [inline src/Service.Host/client/src/components/Root.js]: detail-report/report seems a bit of a repeat

File: src/Service.Host/client/src/components/reports/PartFailDetailsReport.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import Typography from '@material-ui/core/Typography';
+import { ReportTable, Loading, Title, ErrorCard } from '@linn-it/linn-form-components-library';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import Page from '../../containers/Page';
+
+export default function PartFailDetailsReport({ reportData, loading, options, error }) {
+    return (
+        <Page>
+            {console.log(options)}
```
Comment (richardiphillips) [inline src/Service.Host/client/src/components/reports/PartFailDetailsReport.js]: don't need console log

File: src/Service.Host/client/src/containers/reports/PartFailDetailsReport.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import { connect } from 'react-redux';
+import { ReportSelectors, getItemErrorDetailMessage } from '@linn-it/linn-form-components-library';
+import queryString from 'query-string';
+import initialiseOnMount from '../initialiseOnMount';
```
Comment (richardiphillips) [inline src/Service.Host/client/src/containers/reports/PartFailDetailsReport.js]: can use shared library initialiseOnMount now

**Comment by atcampbell** [comment]
> approved
> 
> felt longer than a "a not too big" PR

Cheekily trying to draw you in...

File: src/Service.Host/client/src/components/Root.js (hunk 1 above)
Comment (atcampbell) [reply src/Service.Host/client/src/components/Root.js]: Do you think `/production/quality/part-fails/detail/report` would be better?

**Pull request merged**